runtime/rococo/src/lib.rs

```diff
@@ -1478,8 +1478,9 @@ pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
 pub type Migrations = (
+	pallet_balances::migration::ResetInactive<Runtime>,
 	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckAccount>,
```

Comment:
```suggestion
	// We need to apply this migration again, because `ResetInactive` resets the state again.
	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckAccount>,
```